index.md

```diff
@@ -12,17 +12,29 @@ homepage: true
   </p>
 </div>
 
-Pulp is a platform for managing repositories of software packages and making them available to a
-large number of consumers. Pulp can locally mirror all or part of a repository, host your own
-software packages in repositories, and manage many types of content from multiple sources in one
-place.
+If you have to manage hundreds or thousands of packages, Pulp can help!
+
+![](/images/pulp-workflow-architecture-ha/pulp-101.png)
+
+
+* **Ensure stability and continuity:** External content sources can go offline unexpectedly.
+If you want to ensure that you always have what you need, Pulp can help.
+* **Stop using rsync:** Pulp is designed with complex content management workflows and disk optimization in mind.
+If your sync script is letting you down, Pulp can help.
+* **Reduce rate limiting:** From one day to the next, third party platforms can introduce rate limiting and change the conditions of service.
+If you want to reduce operation costs by having your team consume content from Pulp rather than third parties, Pulp can help.
+* **Distribute content privately:** Sometimes you need a way to distribute private content you have developed in house.
+If you want to keep your private packages off third-party platforms and distribute them with ease, Pulp can help.
+* **Experiment without risk:** Every change to content hosted in Pulp creates a new repository version.
+If you need to pin packages to certain versions to ensure stability and repeatability, Pulp can help.
```

Comment:
```suggestion
If you need to pin packages to certain versions to ensure stability or if you need to rollback changes to repos, Pulp can help.
```

Reply:
Maybe this change doesn't make sense. I was just thinking it'd be nice to call out the ability to rollback changes to repo somehow.

Reply:
I committed it and now see that it reads badly. I love the term repeatability. I'll try weave in rollback.

Reply:
Would this work?